app/src/main/AndroidManifest.xml

```diff
@@ -24,12 +25,16 @@
             </intent-filter>
         </activity>
 
-        <activity android:name="io.appwrite.views.CallbackActivity" >
+        <activity
+            android:name="io.appwrite.views.CallbackActivity"
+            android:exported="true">
             <intent-filter android:label="android_web_auth">
                 <action android:name="android.intent.action.VIEW" />
+
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-                <data android:scheme="appwrite-callback-6070749e6acd4"  />
+
+                <data android:scheme="appwrite-callback-" />
```

Comment:
Let's leave some helper text in place of the ID

```suggestion
                <data android:scheme="appwrite-callback-[PROJECT_ID]" />
```

Reply:
Done